Python/crossinterp.c

```diff
@@ -1687,6 +1687,7 @@ _PyXI_ApplyCapturedException(_PyXI_session *session)
     assert(session->error != NULL);
     PyObject *res = _PyXI_ApplyError(session->error);
     assert((res == NULL) != (PyErr_Occurred() == NULL));
+    _PyXI_excinfo_Clear(&session->error->uncaught);
```

Comment:
This should probably use `xi_error_clear()`:

```suggestion
    xi_error_clear(&session->error);
```

Does that cause any problems?  (It shouldn't.)

Reply:
There’s no `xi_error_clear` function in CPython 3.13; it was only added in 3.14.